apps/frontend/app/(client)/(main)/course/[courseId]/layout.tsx

```diff
@@ -0,0 +1,33 @@
+import { Separator } from '@/components/shadcn/separator'
+import codedangLogo from '@/public/logos/codedang-with-text.svg'
+import Image from 'next/image'
+import Link from 'next/link'
+import { Header } from './_components/Header'
+import { Sidebar } from './_components/Sidebar'
+
+export default function Layout({ children }: { children: React.ReactNode }) {
+  return (
+    <div className="flex h-dvh w-full flex-col">
+      <header>
+        <Header />
+      </header>
```

Comment:
```suggestion
       <Header />
```

중첩 `header` 태그가 생기는 것 같아요!